lib/Screens/account_dashboard.dart

```diff
@@ -20,97 +25,453 @@ class _AccountDashboardState extends State<AccountDashboard> {
+              fontSize: 12,
+              fontWeight: FontWeight.bold,
+              color: LightColor.navyBlue2,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _showAccountChoosingBottomSheet() {
+    final List<Widget> columnWidgets = [
+      const Padding(
+        padding: EdgeInsets.all(
+          20.0,
+        ),
+        child: TitleText(text: 'Accounts'),
+      ),
+      const Divider(
+        height: 20,
+      ),
+    ];
+    final List<Widget> accountButtons =
+        accounts.map((Account acc) => _getAccountTile(context, acc)).toList();
+
+    columnWidgets.addAll(accountButtons);
+
+    showModalBottomSheet<void>(
+      context: context,
+      builder: (context) {
+        return Container(
+          height: 350.0,
+          color: const Color(0xFF737373),
+          child: Container(
+            decoration: const BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.only(
+                topLeft: Radius.circular(
+                  20.0,
+                ),
+                topRight: Radius.circular(
+                  20.0,
+                ),
+              ),
+            ),
+            child: ListView(
+              children: columnWidgets,
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _appBar() {
+    return Row(
+      children: <Widget>[
+        const CircleAvatar(),
+        const SizedBox(width: 15),
+        TitleText(
+          text: account.alias,
+        ),
+        const Expanded(
+          child: SizedBox(),
+        ),
+        GestureDetector(
+          onTap: () {
+            _showAccountChoosingBottomSheet();
+          },
+          child: Icon(
+            Icons.short_text,
+            color: Theme.of(context).iconTheme.color,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _operationsWidget() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: <Widget>[
+        _icon(
+          Icons.transfer_within_a_station,
+          'Transfer',
+        ),
+        _icon(
+          Icons.delete_sweep,
+          'Unlink',
+        ),
+        _icon(
+          Icons.code,
+          'Qr Pay',
+        ),
+      ],
+    );
+  }
+
+  Widget _icon(IconData icon, String text) {
+    return Column(
+      children: <Widget>[
+        GestureDetector(
+          onTap: () {},
+          child: Container(
+            height: 80,
+            width: 80,
+            margin: const EdgeInsets.symmetric(
+              vertical: 10,
+            ),
+            decoration: const BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.all(
+                Radius.circular(
+                  20,
+                ),
+              ),
+              boxShadow: [
+                BoxShadow(
+                  color: Color(0xfff3f3f3),
+                  offset: Offset(
+                    5,
+                    5,
```

Comment:
```suggestion
                    5
```

I think you don't need to put trailing comma for simple parameters like this... because it wastes a lot of lines unnecessarily... this applies to a lot of other parts of the code. Can just use your intuition on places where trailing commas would make the code cleaner :)